app/javascript/controllers/media_wrapper_controller.js

```diff
@@ -11,13 +11,17 @@ export default class extends Controller {
   toggleVisibility(event) {
     const index =  event.detail.index
     this.element.hidden = this.indexValue !== index
-    this.pauseAllMedia();
+    this.pauseAllMedia(index);
   }
 
-  pauseAllMedia() {
+  pauseAllMedia(index) {
```

Comment:
Can you document the purpose of the index parameter?

Reply:
Yep. Done.